build_torch_xla_libs.sh

```diff
@@ -71,6 +71,13 @@ if [ "$CMD" == "clean" ]; then
   bazel clean
   popd
 else
+  # Overlay llvm-raw secondary cache. The remote cache should be updated
+  # nightly with the pinned llvm archive. Note, this commands will be NO-OP if there is no match.
+  sed -i '/.*github.com\/llvm.*,/a "https://storage.googleapis.com/tpu-pytorch/llvm-raw/{commit}.tar.gz".format(commit = LLVM_COMMIT),' \
+    $THIRD_PARTY_DIR/tensorflow/third_party/llvm/workspace.bzl
```

Comment:
so this only get updated when pytorch/xla update its tf pin?

Reply:
This is local update to the build file, it needs to be done before every build. It's local because the change is temporary and specific to each CI run.